test/test-tools-service-file-generator.rb

```diff
@@ -0,0 +1,153 @@
+require_relative "helpers/sandbox"
+
+class ToolsServiceFileGeneratorTestCase < Test::Unit::TestCase
+  include Helpers::CommandRunnable
+
+  sub_test_case "generate-pgroonga-primary-maintainer-service" do
+    command = "tools/systemd/generate-pgroonga-primary-maintainer-service.sh"
+
+    test "default" do
+      output, _ =  run_command(command)
+      assert_equal(<<-EXPECTED, output)
+# How to install:
+#   tools/systemd/generate-pgroonga-primary-maintainer-service.sh | sudo -H tee /lib/systemd/system/pgroonga-primary-maintainer.service
+[Unit]
+Description=PGroonga primary maintainer
+
+[Service]
+Type=oneshot
+User=pgrn
+Group=pgrn
```

Comment:
This will not be portable.

`Etc.getpwuid.name`/`Etc.getgrgid.name` may be better.